### ISSUE TITLE
Add support for new GitHub check suite states

### ISSUE THREAD

**Issue by eseliger** [sourcegraph/sourcegraph#16078]
GitHub added new states, so we should add them to the parsing function here: https://github.com/sourcegraph/sourcegraph/blob/main/enterprise/internal/campaigns/state.go#L335

```
// Schema Changes for 2020-02-05
// Enum value STALE was added to enum CheckConclusionState
// Enum value SKIPPED was added to enum CheckConclusionState
```

**Comment by neel229** [comment]
Hey, I would like to work on this but can someone elaborate?

**Comment by eseliger** [comment]
Hello @neel229 👋 
Thank you for your interest in working on this, unfortunately it has been prioritized for a new hire starting soon, so I cannot assign it yet but it is already taken.

**Comment by neel229** [comment]
No worries :)

**Comment by ibilalkayy** [comment]
Hey, @eseliger can I still work on this issue if it is not resolved?

**Comment by eseliger** [comment]
I don't think it is resolved, but also I haven't heard any problems arise from it so not sure of it's relevance. If you want to take a look, feel free to, we'd be happy to accept PRs :)
